x-pack/plugins/newsfeed/server/routes/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter, HttpResponseOptions } from '../../../../../src/core/server';
+import { fetchNewsItems } from '../lib/fetch_news_items';
+
+interface RouteDefinitionParams {
+  router: IRouter;
+  kibanaVersion: string;
+}
+
+function defineGetItemsRoute({ router, kibanaVersion }: RouteDefinitionParams) {
+  router.get(
+    {
+      path: '/api/newsfeed/news',
+      validate: false,
+      options: { authRequired: true },
```

Comment:
This may be important, since our plugin runs on the Login page, where the browser has no auth to send